### PR TITLE
Update UG feature screenshots and profile

### PR DESCRIPTION
**Pull request by jetnew**
### What this does
This PR updates the User Guide with Allocation feature screenshots and updates my profile.

### How to test

1. Navigate to the docs folder.
2. Run `bundle exec jekyll serve`.

### Notes
N/A

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -117,7 +125,14 @@ Format: `deallocate si/<student_index>`
...
 Examples:
-* `deallocate si/1` deallocates the room for the student at `student_index` 1. 
+* `deallocate si/1` deallocates the room for the student at `student_index` 1.
+
+##### Before deallocation
+Refer to "After allocation" above.
+
+##### After deallocation
+
+<img src="https://user-images.githubusercontent.com/27071473/97580237-0040da00-1a2e-11eb-9337-f3bf29f85bed.png">
```
Comment (mkeoliya) [inline docs/UserGuide.md]: Bro put the image in our `docs/images` folder and use the markdown format to insert it.

**Review by mkeoliya — CHANGES_REQUESTED** [review]
Changed requested.

File: docs/UserGuide.md (hunk 1 above)
Comment (JingYenLoh) [reply docs/UserGuide.md]: Bro it's "change(s)" requested not change**d** requested.

Comment (jetnew) [reply docs/UserGuide.md]: Changed maded

**Review by mkeoliya — APPROVED** [review]

**Pull request merged**